week4tasks/src/week4/reversing/Main.java

```diff
@@ -0,0 +1,30 @@
+package week4.reversing;
+
+import java.util.Scanner;
+
+class reverser{
+     public reverser revesering(String str){
```

Comment:
what is the return type of this method? make sure that your code running without any compilation errors before pushing your code please